src/styles/main.scss

```diff
@@ -0,0 +1,70 @@
+@import 'variables';
+
+body {
+  margin: 0;
+  height: 100vh;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  color: $text-color;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+
+  gap: $border-day-gap;
+  padding: 10px;
+  max-width: ($day-size * 7 + $border-day-gap * 6);
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    height: 100px;
+    width: 100px;
+    background-color: #eee;
+    border: 1px solid $text-color;
+
+    cursor: pointer;
+    transition-duration: 500ms;
+
+    &:hover {
+      background-color: #ffbfcb;
+      transform: translateY(-20px);
+    }
+  }
+
+  .calendar__day::before {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    height: 100px;
+    width: 100px;
+  }
```

Comment:
The pseudo-element `.calendar__day::before` is set to `display: flex`, but it does not have any content or children to justify or align. This might be unnecessary unless you plan to add content inside the pseudo-element.